### PR TITLE
[Cherry-pick] Check compress log MMN

### PR DESCRIPTION
**Pull request by Hikaru2035**
# Ticket
#435 

# General
- [ ] No unnecessary files, debug logs, or commented code included.
- [ ] PR has at least 1 peer approval before senior review.
- [x] I fully understand the code I submitted.
- [ ] No hardcoded configuration or secrets.
- [ ] **Environment Variables** changes? (Leave unchecked if N/A)

# Design & Solution Clarity
- [ ] A design diagram (sequence / flow) is attached or linked.
- [ ] The implementation matches the design.
- [ ] All major flows and decision points are represented in the diagram.
- [ ] Error handling and edge cases are clearly defined in the design.

Design link (diagram / doc):
`<paste link here>`

# Testing
- [ ] Unit tests cover main logic and important edge cases.

> Note: Complex logic must not rely on manual testing only.

# UI evidence (For frontend only)
`<paste screenshots here>`

# Peer Review Checklist
- [ ] I understand the overall solution and flow.
- [ ] The code follows the provided design diagram.
- [ ] No obvious logic or security issues found.
- [ ] Tests are sufficient for the scope of this change.

### CODE REVIEW

File: cmd/init.go [hunk 1]
```diff
@@ -313,7 +320,99 @@ func initializeFileLogger() {
...
+			}
+		}
+	}
+}
+
+func compressLogFile(filePath string) {
+	gzPath := filePath + ".gz"
+
+	// Implementation for compressing log file to gzip
+	fr, err := os.Open(filePath)
```
Comment (vietdangquoc) [inline cmd/init.go:386]: thêm log

File: cmd/init.go [hunk 2]
```diff
@@ -313,7 +320,99 @@ func initializeFileLogger() {
...
+	if err != nil {
```
Comment (vietdangquoc) [inline cmd/init.go:387]: thêm log

File: cmd/init.go [hunk 3]
```diff
@@ -313,7 +320,99 @@ func initializeFileLogger() {
...
+		return
+	}
+	defer fr.Close()
+
+	fw, err := os.Create(gzPath)
+	if err != nil {
```
Comment (vietdangquoc) [inline cmd/init.go:393]: thêm log vào các error
